omnibus/config/templates/updater/datadog-updater-exp.service.erb

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Datadog Updater Experiment
+After=network.target
+OnFailure=datadog-updater.service
+Conflicts=datadog-updater.service
+JobTimeoutSec=3000 #50 minutes
+
+[Service]
+Type=simple
+PIDFile=<%= install_dir %>/run/updater.pid
+User=dd-agent
+EnvironmentFile=-<%= etc_dir %>/environment
+ExecStart=<%= install_dir %>/bin/updater/updater run -p <%= install_dir %>/run/updater.pid
```

Comment:
the newly installed updater will be in another directory
(static link)
/opt/datadog-packages/datadog-updater/V7.XXX
(symlink for experiment)
/opt/datadog-packages/datadog-updater/experiment